testsuite/features/step_definitions/command_steps.rb

```diff
@@ -34,6 +34,11 @@
   $server.run("curl --insecure --location #{url} --output /dev/null")
 end
 
+Then(/^it should be possible to use the HTTP proxy$/) do
+  url = 'https://updates.suse.com/SUSE/Products/SLE-Full/15-SP2/x86_64/iso/SLE-15-SP2-Full-x86_64-GM-Media1.iso.sha256.asc?puBv2yb9SYRD9TSM--G7wNmD-brWw-55ZleEl6oW7d9azO-egbH9I1neECM-Fxe8B1Vd67OxhDd2Ve0oLJ2npLoVjR6L7XvMPOph1pTD1hMYiedac3CO5MV9MzAz7o1nvRePINPUELIGwGe76UpJ8jjz5RdxUio49sBkjBVMv3MfgYnm4Qx-_eNIl9A&_ga=2.74585554.1711402403.1603885305-150982903.1600156280'
```

Comment:
I guess the hash passed by parameter will not expire. If so, all good.

Reply:
That's the question. I wanted something on `updates.suse.com` in order to be as close as possible from what the reposync downloads, but all files there use hashes, or you get a 403.

This hash corresponds to one of the files in public access from SUSE's web site. After a few hours, it has remained valid, but I have no guarantee for next days or more.

If these hashes do not stay, I am afraid we will have to use a server that is not `updates.suse.com`.

Reply:
In fact, I would prefer to use our mirror, as we do everywhere else.
But... in NUE or even externally for Uyuni users we don't use mirror, so we need an alternative.

Reply:
I would use maybe fjust www.suse.com for the test.
The token has an expire date. At some point in time in the future this test will fail because of this and we all wonder what has changed.

Reply:
So, I would have both possibilities in the check, if we use a mirror in the environment, then we check through the mirror, if we don't like in NUE environments, then we use www.suse.com. So in that way, in environments with a mirror we completely de-couple from external links.

Reply:
@mcalmer i did not see your comment. I will just do another PR for head